stable/kong/values.yaml

```diff
@@ -97,7 +97,7 @@ livenessProbe:
     path: "/status"
     port: admin
     scheme: HTTPS
-  initialDelaySeconds: 30
+  initialDelaySeconds: 180
```

Comment:
2 and 3 minutes are extremely high.
@shashiranjan84 has shown concern around this change in one of this past review.